iis-wf/iis-wf-primary/src/test/resources/eu/dnetlib/iis/wf/primary/processing/sampledataproducer/oozie_app/workflow.xml

```diff
@@ -322,6 +322,19 @@
 					<name>output_matched_doc_organizations</name>
 					<value>${workingDir}/exported/matched_affiliation</value>
 				</property>
+                <!-- shared executors configuration for affmatch and citmatch spark algorithms-->
+                <property>
+                    <name>sparkExecutorMemory</name>
+                    <value>1G</value>
+                </property>
+                <property>
+                    <name>sparkExecutorCores</name>
+                    <value>1</value>
+                </property>
+                <property>
+                    <name>sparkExecutorsNumber</name>
```

Comment:
The same here, could we change the sparkExecutorsNumber to sparkNumExecutors to match the prop name as the other vars?